### ISSUE TITLE
Node.js Foundation Modules Team Meeting 2018-08-01

### ISSUE THREAD

**Issue by MylesBorins** [nodejs/modules#161]
## Time

**UTC Wed 01-Aug-2018 19:00 (07:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 01-Aug-2018 12:00 (12:00 PM) |
| US / Mountain | Wed 01-Aug-2018 13:00 (01:00 PM) |
| US / Central  | Wed 01-Aug-2018 14:00 (02:00 PM) |
| US / Eastern  | Wed 01-Aug-2018 15:00 (03:00 PM) |
| London        | Wed 01-Aug-2018 20:00 (08:00 PM) |
| Amsterdam     | Wed 01-Aug-2018 21:00 (09:00 PM) |
| Moscow        | Wed 01-Aug-2018 22:00 (10:00 PM) |
| Chennai       | Thu 02-Aug-2018 00:30 (12:30 AM) |
| Hangzhou      | Thu 02-Aug-2018 03:00 (03:00 AM) |
| Tokyo         | Thu 02-Aug-2018 04:00 (04:00 AM) |
| Sydney        | Thu 02-Aug-2018 05:00 (05:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Modules%20Team+Meeting+2018-08-01&iso=20180801T19
* or http://www.wolframalpha.com/input/?i=07PM+UTC%2C+Aug+01%2C+2018+in+local+time

## Links

* **Minutes Google Doc**: <https://docs.google.com/document/d/11oDyvFSnPp3mkyvr8SwRQTKo4aEu9Z0khIiQKPdT1Yk/edit?usp=sharing>
* _Previous Minutes Google Doc: <https://docs.google.com/document/d/1zruFPTN9dIpvSEkStxo4fkPenXTdhsLEVkcmoKlVmOA/edit>_

## Agenda

Extracted from **modules-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### Approving PRs (2 Minute Timebox)

* Add Sendil Kumar as an observer [#159](https://github.com/nodejs/modules/pull/159)

### Update on Progress (10 Minute Timebox)

* Have presentation on loaders. [#135](https://github.com/nodejs/modules/issues/135)
  - 3 minute timebox
* Create Terminology.md [#158](https://github.com/nodejs/modules/pull/158)
  - 3 minute timebox
* Developer Survey [#85](https://github.com/nodejs/modules/issues/85)
  - 3 minute timebox

### Discussion (45 Minute Timebox)

* Proposals for configuring parse goals of files in --experimental-modules [#160](https://github.com/nodejs/modules/pull/160)
  - 10 minute timebox
* Features adjustments and grouping [#156](https://github.com/nodejs/modules/pull/156)
  - 10 minute timebox
* ESM in .js files proposals [#150](https://github.com/nodejs/modules/pull/150)
  - 10 minute timebox
* transparent-or-not interop [#90](https://github.com/nodejs/modules/issues/90)
  - 15 minute Timbox

## Invited

* Modules team: @nodejs/modules

## Notes

The agenda comes from issues labelled with `modules-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: https://zoom.us/j/656987750
* For those who just want to watch: https://www.youtube.com/c/nodejs+foundation/live

**Comment by GeoffreyBooth** [comment]
FYI #150 and #160 are connected, we should discuss them together.

**Comment by Fishrock123** [comment]
Ok, I dug it up.  From the meeting, my note on the ancient TSC thing I brought up:

Here is the TSC meeting from February 2017 where I brought up concerns raised by folks that teach JavaScript:
https://github.com/nodejs/CTC/blob/master/meetings/2017-02-01.md (the minutes note is small)
The meeting recording link there is actually wrong, so here is the link with the correct timestamp for when I bring this up: https://youtu.be/8Snc5lh0VZQ?t=23m29s

The tl;dr was:
- require returning a promise that is not explicitly exported is troublesome
- esm's largest benefit to newcomers is likely the ability to deal with circular deps

Unfortunately I can't find my exact notes of the meeting that I had with these stakeholders. I have updated the minutes with these links accordingly.
